packages/web-components/src/components/button/button.ts

```diff
@@ -123,6 +123,12 @@ class CDSButton extends HostListenerMixin(FocusMixin(LitElement)) {
     this.openTooltip = false;
   };
 
+  /**
+   * Specify text for the accessibility label of the button
+   */
+  @property({ attribute: 'aria-label' })
+  ariaLabel = '';
```

Comment:
This should be undefined by default I think, rather than an empty string. This is why it's being included in the snapshot when I don't think it should be.
```suggestion
  ariaLabel!: string;
```